ams/server/src/main/java/com/netease/arctic/server/optimizing/maintainer/MixedTableMaintainer.java

```diff
@@ -324,12 +335,16 @@ public BaseTableMaintainer(UnkeyedTable unkeyedTable) {
 
     @Override
     public Set<String> orphanFileCleanNeedToExcludeFiles() {
-      return mergeSets(changeFiles, baseFiles, hiveFiles);
+      return ImmutableSet.<String>builder()
```

Comment:
Implement mergeSets using ImmutableSet?

Reply:
OK, I will extract a method to make the code more concise.

Reply:
I change to using the `Sets#union`.